### ISSUE TITLE
Failing unit tests

### ISSUE THREAD

**Issue by pearguacamole** [ravin-d-27/PyDeepFlow#31]
Unit test are failing for 
- tests/test_activations.py
- tests/test_losses.py
- tests/test_model.py
on the latest main branch

**Comment by ravin-d-27** [comment]
Hi @pearguacamole , those test files are not yet in operation and need to configure them.

**Comment by pearguacamole** [comment]
@ravin-d-27 I looked in the results, while the test were not completely comprehensive the error they were pointing to are important. For example, if you don't give validation tests, the fit function will throw error as it's still printing validation accuracy(Null) in this case. Similarly some of the loss functions are using functions from device class that are not defined.
You can recreate one by simply running `Iris_Multi_Class.py`. I just wanted to know if someone was working on it, didn't want to interfere with their progress by making pull request with my fixes. :-)

**Comment by ravin-d-27** [comment]
@pearguacamole ! Thank you for letting me know. Right now, nobody is working on this. If you are interested, please work on it. Your contribution will help me out !
